lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs

```diff
@@ -650,10 +695,17 @@ generatorWithWid wids =
             <*> genSortOrder
             <*> genRange
             <*> arbitrary
-    , declareGenerator "RemovePendingTx" 4
-        $ RemovePendingTx <$> genId <*> arbitrary
+    -- TODO: Implement mGetTx
+    -- , declareGenerator "GetTx" 3
+    --     $ GetTx <$> genId <*> arbitrary
```

Comment:
Old TODO which can now be done :thinking: ?

Reply:
Still a TODO - we ought to have mGetTx in the qsm tests. I'll add it to the next PR.